src/components/HomePage/index.tsx

```diff
@@ -26,17 +24,14 @@ const HomePage: React.SFC = () => (
     >
       {({ error, data, loading }) => {
         if (canDisplay(data)) {
+          const backgroundImg = data.shop.homepageCollection.backgroundImage;
```

Comment:
You can also get backgroundImg from homepageCollection using es6 destructing: `const { backgroundImg } = data.shop.homepageCollection`